README.md

```diff
@@ -19,27 +19,59 @@ This app demonstrates all of the Windows UI 3 library controls and styles availa
 - **Design & Accessibility Guidance**: Design and accessibility pages help make the gallery a useful developer companion app. 
 - **Version Adaptive Code:** This sample shows how to write version adaptive code so that the app can run on previous versions of Windows while also using the latest capabilities on the most recent verison of Windows.
 
-## Further information
+
+## 👏 Contribute to WinUI Gallery
+
+Any samples or docs improvements you'd like to see? We're always looking for a helping hand. Feel free to file an issue to start the discussion, or even better, create a PR with the change you'd like to see!
+
+Check out these [help wanted](https://github.com/microsoft/WinUI-Gallery/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) issues if you don't know where to start.

```

Comment:
What about the "good first issue" label? Should we support both or should we stick with one?

Reply:
What do you think? both help wanted and good first issue have a lot of overlap. I wonder if it makes sense to just consolidate to one. WinUI Gallery repo is also a lot less complicated than the WinUI2 repo, so a "good first issue" may not be needed?

Reply:
I agree with you, for larger repositories, good first issue might be a good idea. In our case, help wanted is probably enough.